internal/metrics/sources/prometheus/exporter.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) F5, Inc.
```

Comment:
I'd combine and move `exporter.go` and `otel.go` into their own folder like `metrics/exporter/otel`

Reply:
Yeah true, that should be pulled out of the `prometheus` dir. I'm not a huge fan of adding that much nesting though, if we can avoid it.

Reply:
putting it under just `metrics/exporter` probably makes more sense